packages/testlab/src/testlab.ts

```diff
@@ -16,3 +18,29 @@ shouldAsFunction.use((should, assertion) => {
 export const expect = shouldAsFunction;
 export {sinon};
 export {supertest};
+
+export type Client = supertest.SuperTest<supertest.Test>;
+
+/**
+ * Create a SuperTest client connected to an HTTP server listening
+ * on an ephemeral port and calling `handler` to handle incoming requests.
+ * @param handler
+ */
+export function createClientForHandler(
+    handler: (req: http.ServerRequest, res: http.ServerResponse) => void)
+    : Client {
+  const server = http.createServer(handler);
+  return supertest(server);
+}
+
+export interface Server {
```

Comment:
I don't want to reference the full `Server` class from `@loopback/core` because that would create a circular dependency. Also depending on the layout of `node_modules`, the testlab may see a different copy of Server (a different copy of `@loopback/core` package) than the application under the test.